scripts/plot_summary.py

```diff
@@ -145,7 +145,10 @@ def plot_costs():
         df.index.difference(preferred_order)
     )
 
-    new_columns = df.sum().sort_values().index
+    # Sort columns by sum
+    new_columns = (
+        df.columns
+    )  # If you want to sort by values, then use: new_columns = df.columns.sort_values()
```

Comment:
Shouldn't this be:
` )  # If you want to sort by values, then use: new_columns = df.sum().sort_values().index`

Reply:
True! I will change it

Reply:
Shouldn't also the line above it as: new_columns = df.sum().index
I mean only remove the sorting.  You wrote instead: new_columns = (df.columns)

Reply:
> Shouldn't also the line above it as: new_columns = df.sum().index I mean only remove the sorting. You wrote instead: new_columns = (df.columns)

This is not required in this case and works as intended